Sum_Of_Cubes/Sum_Of_Cubes.c

```diff
@@ -0,0 +1,32 @@
+/*
+    The sum of cubes of first N natural numbers is given
```

Comment:
Good work.
In future PRs, please do not indent the comments that you add on the top like this (descriptions).